docs/multicluster/user-guide.md

```diff
@@ -359,12 +362,16 @@ spec:
 
 ## Multi-cluster Gateway Configuration
 
+Multi-cluster Gateways are required to support multi-cluster Service access
+across member clusters. Each member cluster should have one Node be specified as
+its Multi-cluster Gateway. Multi-cluster Service traffic is routed among clusters
```

Comment:
Change it to `multi-cluster Service traffic`?
I am wondering what's the rule to capitalize the `Multi-cluster`? I saw you kept a few `Multi-cluster` no changed.

Reply:
I feel no need to capitalize "multi-cluster Service". No strong preference, but seems most docs are already like this.

Here, "Multi-cluster" is capitalized as it is the beginning of the sentence!

Reply:
OK, I didn't mean this specific `Multi-cluster Gateways`, I saw those `Multi-cluster Gateway` and `Multi-cluster ClusterSet` are not changed, I'd like to understand the logic here so I can handle it in the same way in the future.

Reply:
Ok. I think "Antrea Multi-cluster" is the name of the feature, Multi-cluster Gateway is the name of the component/concept, the same as "Multi-cluster Service API", so we capitalize them. But if we view "multi-cluster" as a regular word to describe "Service", then no need to capitalize.